templates/tease-page-type.twig

```diff
@@ -25,14 +23,10 @@
 		<div class="search-result-item-info">
 			{% if ( post.author ) %}
 				<span class="search-result-item-author">{{ __( 'by') }}
-					{% if not ( post.author_override ) %}
-						<a href="{{ post.author.path }}">{{ post.author }}</a>
-					{% else %}
-						{{ post.author }}
-					{% endif %}
+					<a href="{{ post.author.path }}">{{ post.author }}</a>
```

Comment:
@pablocubico - if author_override is true , then it will show blank anchor tag.

Reply:
@sagarsdeshmukh It will show the overridden author's name! Have you seen this new FakeUser class I made to provide the same interface as User in case the author is overridden:

https://github.com/greenpeace/planet4-master-theme/pull/618/files/5113d3e51a5f080ae4fe0c53da7532184613241a#diff-6477e98b5e10b503efc1eb59f498b245R249

Reply:
I mean if the author name is overridden, then the author name should not lead to any author profile[no need of anchor tag with overridden author name].
![image](https://user-images.githubusercontent.com/5357471/48554765-4e25ab80-e905-11e8-9b4c-ee5148dd3788.png)